src/core/urdb.jl

```diff
@@ -12,15 +12,15 @@ Contains some of the data for ElectricTariff
 """
 struct URDBrate
     energy_rates::Array{Float64,2}  # time X tier
-    energy_tier_limits::Array{Real,1}
+    energy_tier_limits::Array{Float64,2}
```

Comment:
@zolanaj could you add a comment explaining the dimensions as there is for `energy_rates` here and under `struct ElectricTariff`?

Reply:
> @zolanaj could you add a comment explaining the dimensions (as there is for `energy_rates`) here and under `struct ElectricTariff`?

addressed here: https://github.com/NREL/REopt.jl/commit/f1301b2537993b2230d17af0ea71ca0a02b73166